types/src/front/api_handlers/public/vaults.ts

```diff
@@ -32,25 +32,41 @@ export type PatchDataSourceViewType = t.TypeOf<
   typeof PatchDataSourceViewSchema
 >;
 
-export const PostVaultRequestBodySchema = t.type({
-  name: t.string,
-  memberIds: t.union([t.array(t.string), t.undefined]),
+const PostRestrictedVault = t.type({
```

Comment:
note: it's in "public/vaults" but used in private endpoint. Can you move ?

Reply:
Cleaned up.